src/gpl/src/nesterovPlace.h

```diff
@@ -69,6 +69,9 @@ class NesterovPlaceVars {
 
   int routabilityMaxBloatIter;
   int routabilityMaxInflationIter;
+  
+  int maxRecursionWlCoef;
+  int maxRecursionInitSLPCoef;
```

Comment:
Should these be static const ?

Reply:
Yes, these should. I changed these and commit will be updated to this PR soon